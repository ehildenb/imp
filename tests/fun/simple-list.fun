// [ 3 : 4 : 17 : 8 : [ ] ]

let x = 7
 in [ 3 : 4 : x + 10 : 8 : [ ] ]
