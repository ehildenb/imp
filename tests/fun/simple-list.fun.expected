<FUN>
  <k>
    [ 3 : 4 : 17 : 8 : .Vals ]
  </k>
  <env>
    .Map
  </env>
  <envs>
    .List
  </envs>
</FUN>
